locallibs/fix.py

```diff
@@ -65,10 +65,13 @@ def is_framework_shebang(framework_path, text):
     this_framework_shebang = (
         b"#!" + os.path.abspath(framework_path).encode("UTF-8"))
     default_framework_shebang = b"#!/Library/Frameworks/Python.framework"
+    xcode_shebang = b"#!/Library/Developer/CommandLineTools/usr/bin/python3"
```

Comment:
I wonder what happens if you don't have the CommandLineTools installed but do have a "full" Xcode install? Is there another shebang we need to worry about?

Reply:
I have both cmd line tools and full xcode. Only ever saw this shebang.

Reply:
But what if you only have Xcode and no command line tools? Then I'd think `/Library/Developer/CommandLineTools/usr/bin/python3` wouldn't _exist_.

Reply:
...and verified on a test Mac with no command-line tools install but Xcode 13 present.

Reply:
87% of statistics are made up

Reply:
> 87% of statistics are made up

Truth.

Reply:
Where would I look for that shebang?

Reply:
Easy one I found is using pylint.

Use whichever `python3` you're targeting then use `-s -m pip install pylint` and see where it shoves it.

Reply:
```
$ head ./Python.framework/Versions/Current/bin/pylint
#!/Users/Shared/relocatable-python/Python.framework/Versions/3.9/bin/python3
# -*- coding: utf-8 -*-
import re
import sys
from pylint import run_pylint
if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])
    sys.exit(run_pylint())
```

Reply:
`./Python.framework/Versions/Current/bin/python3 -s -m pip install pylint`

Is this what you are asking?

Reply:
` /Applications/Xcode.app/Contents/Developer/usr/bin/python3 -s -m pip install pylint` for a non-cmd line tool, nor python framework install.

Reply:
I'm confused. In what case would `make_relocatable_python_framework.py` call `/Applications/Xcode.app/Contents/Developer/usr/bin/python3 -s -m pip install foo`? I feel like I'm missing something important here.